pkg/config/config.go

```diff
@@ -199,3 +206,27 @@ func GetCheckById(cfg *Config, id string) *Check {
 	}
 	return nil
 }
+
+func CheckPolicies(cfg *Config, l log.Logger) error {
+	if cfg.Conf.Policy == "" {
+		return nil
+	}
+	for _, p := range cfg.Policies {
```

Comment:
A policy applies to all targets with an AssetType compatible with the CheckTypes referenced by the policy. Now the only thing this function does is check if the policy exists and in this case ignore the `checks` section in the configuration file, as we discussed. Done here [3f4a435](https://github.com/adevinta/vulcan-local/pull/5/commits/3f4a43539849c1aa76761544b7bce2c8393430c6).

Reply:
I would move the code of this function right into the caller function in main.go:
* It's very simple
* It's kind of unclear that a method CheckPolicies does this removing the checks and I think it would improve the readability of the code if it's in the main code.

Reply:
CheckPolicies function has been deleted.